### PR TITLE
Fix isStageNode function

### PR DESCRIPTION
**Pull request by zuzuviewer**
Fix isStageNode function

### CODE REVIEW

File: rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java [hunk 1]
```diff
@@ -58,7 +60,18 @@ public void setStageFlowNodes(List<AtomFlowNodeExt> stageFlowNodes) {
     }
 
     public static boolean isStageNode(FlowNode node) {
-        return (node.getAction(LabelAction.class) != null && node.getAction(ThreadNameAction.class) == null);
+        final boolean stageNode = (node.getAction(LabelAction.class) != null && node.getAction(ThreadNameAction.class) == null);
```
Comment (olamy) [inline rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java:63]: no need of final.

**Review by olamy — CHANGES_REQUESTED** [review]

File: rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java (hunk 1 above)
Comment (zuzuviewer) [reply rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java:63]: > no need of final.

Why？

Comment (olamy) [reply rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java:63]: well I could ask why adding final for a local variable?
frankly it doesn't change anything nor adding any value except adding more (useless?) keyword :) 
if you think it's very important and improve performance or anything else I will not complain.
but I don't see the real added value.
well maybe it's nit picking :)

Comment (zuzuviewer) [reply rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java:63]: Ok, I think it's a good way to tell other people this is unmodifiable variable, and don't try to change it's value, and this is my habit.
I agress with you it is not very important in here , but it's not a bad thing.
So if you still think it's no need, I will remove it;

Comment (olamy) [reply rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java:63]: > Ok, I think it's a good way to tell other people this is unmodifiable variable, and don't try to change it's value, and this is my habit. I agress with you it is not very important in here , but it's not a bad thing. So if you still think it's no need, I will remove it;

why not but why here?
it's a local variable of a very small method use only once in the method body. 
So you probably don't even need to use a local variable.

I cannot find any unit test for this change.
Is it something you could add please?

Comment (zuzuviewer) [reply rest-api/src/main/java/com/cloudbees/workflow/rest/external/StageNodeExt.java]: I add a test for it

**Pull request closed without merging**